idearly/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from "@chakra-ui/icons";
+import * as S from "./Pagination.styles";
+import { PropsWithChildren } from "react";
+import type { IPagiantion } from "./Pagination.types";
+
+export const Pagination = ({
+  currentPage,
+  setCurrentPage,
+  itemsPerPage,
+  dataLength,
+}: PropsWithChildren<IPagiantion>) => {
+  const pageNumbers = [];
+  for (let i = 0; i < Math.ceil(dataLength / itemsPerPage); i++) {
+    pageNumbers.push(i);
+  }
+
+  const handlePaginationClick = (cur: number | string) => {
+    if (typeof cur === "number") {
+      setCurrentPage(cur);
+    } else {
+      switch (cur) {
+        case "prevPage":
+          setCurrentPage((prev) => (prev >= 1 ? prev - 1 : prev));
+          break;
+        case "nextPage":
+          setCurrentPage((prev) =>
+            prev < pageNumbers.length - 1 ? prev + 1 : prev
+          );
+          break;
+        case "firstPage":
+          setCurrentPage(0);
+          break;
+        case "lastPage":
+          setCurrentPage(pageNumbers.length - 1);
+          break;
+      }
+    }
+    window.scrollTo(0, 0);
+  };
+
+  return (
+    <S.PaginationWrapper>
+      <S.PaginationContainer>
+        <S.PaginationIconButton
+          isRound={true}
+          variant="solid"
+          colorScheme="teal"
+          size="xs"
+          aria-label="Done"
```

Comment:
aria-label이 모두 동일하게 되어있는데 요 부분은 각자 설명에 맞게 바꾸는건 어떨까요?

Reply:
오 어떤 수정을 말씀하시는 건지 나중에 설명해주실수 있을까요??

Reply:
`IconButton composes the Button component except that it renders only an icon. Since IconButton only renders an icon, you must pass the aria-label prop, so screen readers can give meaning to the button.`

Chakra ui 공식문서에 이렇게 나와있더라구요! 화면 판독기를 위해서 라벨에 설명을 적어달라고 되어있어 required prop이라 next, prev 이런식으로 바꾸는것도 좋을 것 같은 생각입니다!

하지만 바꾸지 않아도 큰 상관은 없을 것 같아요~!